client/community/action-creators/dns-control/async-check-hosted-zone.js

```diff
@@ -0,0 +1,20 @@
+import * as t from '../../action-types'
+import { createAction } from '../create-action'
+import AuthSelectors from '~client/account/redux/selectors'
+import * as CommunitySelectors from '../../selectors'
+
+export default dnsHostedZone => (dispatch, getState, { api }) => {
+  const credentials = AuthSelectors(getState()).getCredentials()
+  const community = CommunitySelectors.getCurrent(getState())
+  dispatch(createAction(t.CHECK_DNS_HOSTED_ZONE_REQUEST))
+  return api
+    .get(`/communities/${community.id}/dns_hosted_zones/${dnsHostedZone.id}/check`, { headers: credentials })
```

Comment:
We could keep the length of the line with no more than 100 chars, what do you think? The right length is 80 chars as mentioned [in this article](http://www.codereadability.com/maximum-line-length/).

This line could be "splited" like this:
```js
api.get(
  `/communities/${community.id}/dns_hosted_zones/${dnsHostedZone.id}/check`,
  { headers: credentials }
)
```

Or, in my opinion, would be better if we specify a "named variable" that makes more explicit what those data mean like:
```js
const endpoint = `/communities/${community.id}/dns_hosted_zones/${dnsHostedZone.id}/check`
const config = { headers: credentials }
api.get(endpoint, config)
```

That makes it more **vertical** code instead of **horizontal**

Just some coding style tips. What your opinion about some coding style?